inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/include/detection_output_common.cl

```diff
@@ -45,9 +45,11 @@
 // Number of bboxes to keep in output
 #define KEEP_BBOXES_NUM ((KEEP_TOP_K < NUM_OF_IMAGE_BBOXES)? KEEP_TOP_K : NUM_OF_IMAGE_BBOXES)
 
-void FUNC(get_decoded_bbox)(UNIT_TYPE* decoded_bbox, __global UNIT_TYPE* input_location, __global UNIT_TYPE* input_prior_box, const uint idx_prior, const uint idx_class, const uint idx_image)
+inline void FUNC(get_decoded_bbox)(UNIT_TYPE* decoded_bbox, __global UNIT_TYPE* input_location, __global UNIT_TYPE* input_prior_box, const uint idx_prior, const uint idx_class, const uint idx_image)
```

Comment:
I think we'd better separate this function according to its conditions so that we could save compile time and also reduce the branching and code size.